packages/components/src/components/text-input/_text-input.scss

```diff
@@ -64,6 +64,28 @@
     background-color: $field-02;
   }
 
+  //-----------------------------
+  // Fluid Text Input
+  //-----------------------------
+  .#{$prefix}--text-input-wrapper--fluid {
+    position: relative;
+  }
+  .#{$prefix}--text-input-wrapper--fluid .#{$prefix}--label {
+    position: absolute;
+    top: rem(14px);
+    left: rem(14px);
```

Comment:
```suggestion
    top: rem(13px);
    left: $carbon--spacing-05;
```

Reply:
the reason I made this 14 instead of 16 is because for some reason if I put 16 it goes past aligning with the placeholder.

Reply:
It shouldn't if you adjust the text input padding to 16 as well, see https://github.com/carbon-design-system/carbon/pull/5830/files#r406366436

<img width="1005" alt="Screen Shot 2020-04-09 at 11 55 26 AM" src="https://user-images.githubusercontent.com/11928039/78930518-1dde3000-7a59-11ea-9f0a-1b22d0c6e807.png">